clients/src/main/java/org/apache/kafka/common/record/SimplifiedDefaultRecord.java

```diff
@@ -0,0 +1,141 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.kafka.common.record;
+
+import org.apache.kafka.common.header.Header;
+
+import java.nio.ByteBuffer;
+
+import static org.apache.kafka.common.record.RecordBatch.MAGIC_VALUE_V2;
+
+/**
+ *
+ * @Flower.min
+ */
+public class SimplifiedDefaultRecord implements Record {
```

Comment:
I'd suggest letting this class extend from DefaultRecord instead of implements Record.

Also nit: how about rename the `simplified` to `skipKV`, i.e. `SkipKVDefaultRecord`, `SkipKVIterator` etc, to be more explicit.